### PR TITLE
set external_ids:ovn-set-local-ip="true" on nodes 

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
When we do not set external_ids:ovn-set-local-ip="true" on OVS we doe not set local_ip parameter when creating tunnel ports and ipv6 does not work with multiple addresses on the same interface 

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by JacobTanenbaum** [comment]
the test failure has been identified as a flake in issue https://github.com/ovn-org/ovn-kubernetes/issues/4289

File: go-controller/pkg/node/default_node_network_controller.go [hunk 1]
```diff
@@ -308,6 +308,8 @@ func setupOVNNode(node *kapi.Node) error {
 		fmt.Sprintf("external_ids:ovn-monitor-all=%t", config.Default.MonitorAll),
 		fmt.Sprintf("external_ids:ovn-ofctrl-wait-before-clear=%d", config.Default.OfctrlWaitBeforeClear),
 		fmt.Sprintf("external_ids:ovn-enable-lflow-cache=%t", config.Default.LFlowCacheEnable),
+		// when creating tunnel ports set local_ip, helps ensures multiple interfaces and ipv6 will work
+		"external_ids:ovn-set-local-ip=\"true\"",
```
Comment (tssurya) [inline go-controller/pkg/node/default_node_network_controller.go:312]: @JacobTanenbaum I see:
```
external_ids:ovn-set-local-ip
                     The boolean flag indicates if ovn-controller when  create
                     tunnel  ports should set local_ip parameter. Can be hepl‐
                     ful to pin source outer IP for the tunnel  when  multiple
                     interfaces are used on the host for overlay traffic. This
                     is  also  useful when running multiple ovn-controller in‐
                     stances on the same chassis, in which case  this  setting
                     will  guarantee  that their tunnel ports have unique con‐
                     figuration and can exist in parallel.
```

1. when  multiple interfaces are used on the host for overlay traffic. => we don't do this..
2. This is  also  useful when running multiple ovn-controller instances on the same chassis => we don't do this too
3. finally the stupid question ; is this inferring a pinning to primary ip somehow? we are just passing a boolean here right? how does it pick which IP to pin to?

Comment (tssurya) [reply go-controller/pkg/node/default_node_network_controller.go:312]: got questions answered in slack all good.

**Review by tssurya — APPROVED** [review]
/lgtm

**Pull request merged**